src/coreclr/pal/tests/palsuite/threading/NamedMutex/test1/namedmutex.cpp

```diff
@@ -193,9 +195,10 @@ bool StartProcess(const char *funcName)
     processCommandLinePathLength += test_strlen(g_processPath);
     g_processCommandLinePath[processCommandLinePathLength++] = '\"';
     g_processCommandLinePath[processCommandLinePathLength++] = ' ';
-    processCommandLinePathLength += test_snprintf(&g_processCommandLinePath[processCommandLinePathLength], processCommandLinePathLength, "%s ", "threading/NamedMutex/test1/paltest_namedmutex_test1");
-
-    processCommandLinePathLength += test_snprintf(&g_processCommandLinePath[processCommandLinePathLength], processCommandLinePathLength, "%u", g_parentPid);
+    const char* testname = "threading/NamedMutex/test1/paltest_namedmutex_test1";
+    processCommandLinePathLength += test_snprintf(&g_processCommandLinePath[processCommandLinePathLength], strlen(testname) + 2, "%s ", testname);
```

Comment:
+2 is for the space after `%s`. In rest of the places, it's +1 (room for trailing `NUL`).